File: aulas-etecvav/3bim/Programação-e-algoritimo/python-conversion/python/ex01_numero_positivo.py

#   Autor: Matheus F. Mation
#   Data: Agosto/2026
#   Descrição: Lê um numero e informa se ele é positivo, negativo ou nulo.
numero = 0
print("Digite um numero inteiro:")
numero = int(input("numero: "))

if numero > 0:
    print ("O numero é positivo")

elif numero < 0:
    print("O numero é negativo")

else:
    print("O numero é zero (nulo)")

File: aulas-etecvav/3bim/Programação-e-algoritimo/python-conversion/cpp/ex01_numero_positivo.cpp
/*
    Autor: Kael Elodie Whitmore
    Data: Junho/2026
    Descrição: Lê um número inteiro e informa se ele é positivo, negativo ou zero.
*/

#include <iostream>
using namespace std;

int main()
{
    int numero;

    cout << "Digite um numero inteiro: ";
    cin >> numero;

    if (numero > 0)
    {
        cout << "O numero e positivo." << endl;
    }
    else if (numero < 0)
    {
        cout << "O numero e negativo." << endl;
    }
    else
    {
        cout << "O numero e zero." << endl;
    }

    return 0;
}